mlflow/pytorch/__init__.py

```diff
@@ -64,10 +64,10 @@ def get_default_conda_env():
 
 
 def log_model(pytorch_model, artifact_path, conda_env=None, code_paths=None,
-              pickle_module=mlflow_pytorch_pickle_module, **kwargs):
+              pickle_module=None, **kwargs):
     """
     log_model(pytorch_model, artifact_path, conda_env=None, code_paths=None,\
-              pickle_module=mlflow.pytorch.pickle_module, **kwargs)
+              pickle_module=None, **kwargs)
 
```

Comment:
This can probably be removed. It was added to fix the same doc generation problem.

Reply:
Oh, I see.